deps/v8/gypfiles/toolchain.gypi

```diff
@@ -1134,121 +1134,7 @@
       }],
     ],  # conditions
     'configurations': {
-      # Abstract configuration for v8_optimized_debug == 0.
```

Comment:
If I am reading this correctly, this switches the configuration-based conditional to a variable-based conditional (`v8_optimized_debug`) to fix the bug? Can you paste the PR message into the commit message of the second commit, since it is a bit difficult to tell what the diff is doing due to all the indentations?

Reply:
the tl;dr if I flattened the inheritance structure. form:
https://github.com/nodejs/node/blob/9800fd7aaa7226fff3f21edd7f77f563b9a69753/deps/v8/gypfiles/toolchain.gypi#L1315-L1325
to
https://github.com/nodejs/node/blob/693c7bbe621f4ab8686397e301e1f8027e734dff/deps/v8/gypfiles/toolchain.gypi#L1136-L1137
...
https://github.com/nodejs/node/blob/693c7bbe621f4ab8686397e301e1f8027e734dff/deps/v8/gypfiles/toolchain.gypi#L1200

I'll document that